manager/controllers/default/security/profile.class.php

```diff
@@ -57,7 +57,11 @@ public function process(array $scriptProperties = array()) {}
      * @return string
      */
     public function getPageTitle() {
-        return $this->modx->lexicon('profile');
+        if($this->modx->user == null) {
+            return $this->modx->lexicon('user_err_nf');
+        } else {
+            return $this->modx->lexicon('profile').': '.$this->modx->user->get('username');
```

Comment:
Also needs htmlentities around the username.

Reply:
Done